packages/devops/ci/testing.env.encrypted

```diff
@@ -1,2 +1,2 @@
 codeship:v2
-1G4wnRQPEBGB4/MZsd1RgUp9B1ipE2LPs8Xq2Y9JUHZ8V5tUGXCwWXKrXZAjocoVZKIU+60+6675ueg2aSg1KeRNPOC/31egpBAgjnOtDjYRjgIbH5vAaU0HiLqsbeAFzhI23O0Ap4URNUuqWSo8cpu5gt4smlrVyrsQ1xuJPdLvtnOar91Bbbyp5MvJIg4+wV6MgY2iLnwQIqSWxvGn1C+MMyWlSjGTIlwezxVxTHJ97cGRcd+Oym151XqnpVHPrRHLTcLm7b4F4S0xmjcjtyl9pWPsgHg5+Bo28TKCFS67PI7Egrx3aWnZA6WlnB/vC5EdZwfoEEdYWzwUKH7T/ssCM2+JOlHIAGb+NPYwX1SDJ5nG0WZyhPoKVGoxAqPOJExgYPZ2Pf1qOb5ldVVVtFsKF0vBMfhMHMr9L/VcF1K+SXrWu2WvgnsV7FwLH7GvrQyQuOAp4ciEKa3VyCBLzsP0xX4DGj9dfEb8i8diCnjGm8UWpr6T09BOSBzE6jzn8HL4V6WKQquo7ziMHvhjAKUL3zl1nnk+vcBgDIQrVZvuLIYiH0FOmTed9ytKE5tIsmkua04hcaSbVOnWIiY6OWkWvPxYY7MsnENxOtHDAEYGlZPLJAtja5/ilcTpK4WMqz57wm7HoLn+kzy0w/7jpLcUrF29VTEOdrzwmbq3PASZYdRR0+fzido=
\ No newline at end of file
+Xe0EbawQHSultl2ATorXCdSl5h/yWZie0B28nlSLBSl9NJ5VNFo0QTq146p/LjJagfL0+My5ZGTHfJ3vVf6E9+2eo6xn/z59xK+RtcD61w7uFup2Z5ocqTez5aXK6K2qZbyWmdSyY1cDyXR9gsQWtUU9rakh7SyQgxZRwez8ZfU2SdS+Iglpfc/77SNzYVGMnwJb27/h93Bb+qvXeOsgbG+J05/1FEoFD3Xn3KIZmC6nZFWHMc0SHKjVLkrrTXnxrGQvl7VocY7dWC4K80sH7hnGhVDpA2e3v5K9BqDQv7N0EgiGqvueuugId+l1rlTCKQ4+9LBK18V3cU9em6DG1/7dB9sReCJSa5LEPMStbPQQ7ubgx2G2vRIL3Pci/PWmyBkGDfNy417PrTMpdln+Wnmy96jQ0+N8yEo1cwmu/L2WUs2NTyNkLVibVlhL6q2YpwuNjTqNJwzPV/7pmXHPdSfybmW+l5gD5y53jZtnJa8VC4VsxOQS4Q9HUfry69nTVK59BkVTaRRR1/FmPVmIDUkxTp4crwG0Y6xU4+o073yYZ8TXWemeCY64jXeOUWECcbltKSZKqr8102Gbt/trwSo3Kh8bY2rZPYpp/otlk9/9HIQYAP6zjaX19owNXPtlpFGSkH1sXLCZcMrVT4q5MFZ6qv5A4fpeptWqog1ZrRy44JEGSuOsERrBDgfPJESKM6Aul5A+uwW5DuvJ60aVhNSs3OYApPcx1XdbXEMqup1IdTtfgsE0AK3WlLcHLmyyeNMcbBHGlzme4Gvbo0NQpwK3VXbfprB4AjOigFirV2ThprBAnPO+7cs6DYkgHPKBgfNs7pDrFSER5pWD
```

Comment:
Does it change automatically? Like yarn.lock changes after running `yarn`?

Reply:
No, we actually have to manually update this using `jet encrypt`. Read [here](https://beyond-essential.slab.com/posts/ci-cd-using-codeship-uzxspw8z#hht9d-ci-cd-process-secrets) for more info